build-support/bin/release_helper.py

```diff
@@ -787,6 +788,16 @@ def upload_wheels_via_twine() -> None:
     )
 
 
+def prompt_apple_silicon() -> None:
+    input(
+        f"We need to release for Apple Silicon. Please message Eric on Slack asking to release "
```

Comment:
LOL, painting yourself into a corner here... Hopefully this is just temporary.

Reply:
Who else could do this? Chris, at least, right?

Reply:
Yep, although he's not yet a maintainer with publish access.